docs/sources/developer/Developing-Ubuntu.md

```diff
@@ -0,0 +1,120 @@
+# Developing on Ubuntu
+
+## Git
+To download the latest version of Plaso you'll need to install the git tools:
+```
+sudo apt-get install git
+```
+
+Checkout the Plaso source from the git repo:
+```
+git clone https://github.com/log2timeline/plaso.git
+```
+
+## Dependencies
+
+Install the dependencies, using either:
+* A [virtualenv](Developing-Virtualenv.html#Fedora%20Core)
+* The [pre-packaged dependencies](Dependencies.html#Fedora%20Core).
```

Comment:
same comment about pre-packaged, please use prepackaged or change everything to pre-packaged

Reply:
Done